src/index.tsx

```diff
@@ -1,6 +1,17 @@
-import React, { useState, useEffect, useRef, useCallback } from 'react'
+import React, {
+  useState,
+  useEffect,
+  useRef,
+  useCallback,
+  Children
+} from 'react'
+import ReactDOMServer from 'react-dom/server'
 import { largestRect } from 'rect-scaler'
 
+const isChildNotNull = (child: React.ReactNode) => {
+  return Boolean(ReactDOMServer.renderToStaticMarkup(<>{child}</>))
+}
+
```

Comment:
why would we need to render to string? seems odd to add a dependency on `react-dom/server` inside this client component.

Reply:
I told you why I did it. I was trying to find a way to remove wrappers around components which don't render a DOM node. This all started from [this question](https://stackoverflow.com/questions/75390182/react-packed-grid-null-child-is-inserting-a-grid-item?noredirect=1#comment133026926_75390182). I'm tao in that conversation.